examples/csv-io/csv_io.bal

```diff
@@ -35,19 +37,20 @@ function main(string... args) {
     string srcFileName = "./files/sample.csv";
     // The record separator of the `.CSV` file is a
     // new line, and the field separator is a comma (,).
-    //io:CSVChannel csvChannel = io:openCsvFile(srcFileName, mode="r",fieldSeparator=",",charset="UTF-8",skipHeaders=0);
     io:CSVChannel csvChannel = io:openCsvFile(srcFileName);
     try {
         io:println("Start processing the CSV file from ", srcFileName);
         process(csvChannel);
         io:println("Processing completed.");
-    } catch (error err) {
-        io:println("An error occurred while processing the records. ", err.message);
+    } catch (error e) {
+        log:printError("An error occurred while processing the records. ",
```

Comment:
"records. " should be changed to "records: "

Reply:
done https://github.com/ballerina-platform/ballerina-examples/pull/621/commits/91a451427d42ac94abe0e8fb99215cf2a8770fc1